docs_devel/docs/07.ManualContainerTaskBuild.md

```diff
@@ -315,6 +263,8 @@ The following fonts must be installed:
 
 # Usage
 
+-> Is this part still relevant?
+
 All the scripts require the language folder as argument and must be run from the
 `doc_src` directory.
 
```

Comment:
I think this one also applies to building manually without a container.
@miurahr Can you confirm this?

Reply:
I moved these clauses from `doc_src/Readme.md`  I do not check these descriptions, it can be obsoleted.